<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QDialog" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Product_add_code_entry">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Product_add_price_entry">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Product_add_Name_entry">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Adding_product_confirm_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color:
	color: qlineargradient(spread:pad, x1:0.381216, y1:0.937364, x2:0.978, y2:0, stop:0.209945 rgba(119, 129, 130, 255), stop:1 rgba(255, 255, 255, 255));
color: black;

}
QPushButton:pressed {

background-color:qlineargradient(spread:pad, x1:0.607735, y1:0.443045, x2:0.110597, y2:0.903, stop:0.209945 rgba(119, 129, 130, 255), stop:1 rgba(255, 255, 255, 255));
color:white;

}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="Product_code_label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 350 11pt &quot;Yu Gothic UI Semilight&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Code :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Product_Name_label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 350 11pt &quot;Yu Gothic UI Semilight&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Price :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Product_Rack_label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 350 11pt &quot;Yu Gothic UI Semilight&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
